### PR TITLE
Improvements

### PR DESCRIPTION
**Pull request by petrparolek**
- Makefile: project
- rework htaccess
- used latest stable versions of composer packages

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -14,8 +31,5 @@ phpstan:
 tests:
 	vendor/bin/tester -s -p php --colors 1 -C tests
 
-tests-coverage:
+coverage:
 	vendor/bin/tester -s -p phpdbg --colors 1 -C --coverage ./coverage.xml --coverage-src ./app tests
-
-dev:
```
Comment (f3l1x) [inline Makefile:20]: Dev chybi?

File: composer.json [hunk 2]
```diff
@@ -19,16 +19,16 @@
     "contributte/bootstrap": "^0.5.0",
     "contributte/cache": "^0.6.0",
     "contributte/di": "^0.5.0",
-    "contributte/http": "^0.5.0",
```
Comment (f3l1x) [inline composer.json:22]: Proc je tu downgrade?

File: Makefile [hunk 3]
```diff
@@ -1,11 +1,28 @@
...
+# DEVELOPMENT ##############################################
+############################################################
 
 qa: cs phpstan
 
 cs:
 	vendor/bin/codesniffer app tests
 
-cfx:
+csfix:
```
[the remark this answers is not in the document]
Comment (petrparolek) [reply Makefile:25]: Máš to tak všude

File: composer.json (hunk 2 above)
Comment (petrparolek) [reply composer.json:22]: protože 0.6.x není stabilní

File: Makefile (hunk 1 above)
Comment (petrparolek) [reply Makefile:20]: OK

File: Makefile [hunk 4]
```diff
@@ -1,11 +1,28 @@
...
+############################################################
+
+project: install setup
+
+install:
+	composer install
+
+setup:
+	mkdir -p var/{tmp,log}
+	chmod +0777 var/{tmp,log}
```
Comment (paveljanda) [inline Makefile:14]: Wy do we have those lines in makefile?

File: www/.htaccess [hunk 5]
```diff
@@ -1,28 +1,20 @@
...
 
-# enable gzip compression
-<IfModule mod_deflate.c>
-	<IfModule mod_filter.c>
-		AddOutputFilterByType DEFLATE text/html text/plain text/xml text/css application/javascript application/json application/xml image/svg+xml
-	</IfModule>
+    # Front controller
+    RewriteCond %{REQUEST_FILENAME} !-f
+    RewriteCond %{REQUEST_FILENAME} !-d
+    RewriteRule !\.(pdf|js|ico|gif|jpg|png|css|rar|zip|tar\.gz|map)$ index.php [L]
```
Comment (paveljanda) [inline www/.htaccess:19]: Why is there spaces instead of tabs?

**Review by paveljanda — CHANGES_REQUESTED** [review]

File: Makefile [hunk 6]
```diff
@@ -1,11 +1,28 @@
...
+
+############################################################
+# DEVELOPMENT ##############################################
+############################################################
```
Comment (paveljanda) [inline Makefile:18]: Pls remove those hashes

Comment (petrparolek) [reply Makefile]: OK

File: Makefile (hunk 4 above)
Comment (petrparolek) [reply Makefile]: OK, removed

File: www/.htaccess (hunk 5 above)
Comment (petrparolek) [reply www/.htaccess]: thx fixed

File: Makefile [hunk 7]
```diff
@@ -1,11 +1,20 @@
-.PHONY: qa cs cfx phpstan tests build
+.PHONY: project install setup qa dev cs csf phpstan tests build
+
+############################################################
+# PROJECT ##################################################
+############################################################
+
+project: install setup
```
Comment (paveljanda) [inline Makefile:7]: I think this stage is not needed anymore as there is no `setup` stage. 👍

Comment (f3l1x) [reply Makefile:7]: There must be setup phase, for deployment. 

```
setup:
	mkdir -p temp log
	chmod +0777 temp log
```

Comment (petrparolek) [reply Makefile:7]: why, it isn't in another skeletons?

**Comment by f3l1x** [comment]
Updated by myself.

**Pull request closed without merging**